ansible/idr-centos-init.yml

```diff
@@ -4,7 +4,16 @@
 - hosts: "{{ idr_environment | default('idr') }}-hosts"
 
   tasks:
-  - name: Create persistent journald directory
+  - name: centos-init | Install locate
+    become: yes
+    yum:
+      name: mlocate
```

Comment:
What's the background for this? The PR openmicroscopy/openmicroscopy#5684 in the description doesn't mention it.

Reply:
It's not an OMERO-specific request but an "IDR devops" request. i.e. for the likes of Mark to better find things on these servers.

Reply:
From a "devops" perspective we shouldn't really need this since it means we haven't defined/documented where everything is. Does it need to be on every server, or just the OMERO ones?

Reply:
OMERO and Database was the initial requirement.

Reply:
Can you make it specific to those then? `idr-00-preinstall.yml` is run by everything- publicidr, ftp, and the k8s vae.

Reply:
Added `idr_debug` after discussion in `#idr`.

Note: also dropped the cron section since @mtbc pointed out that a `daily` job is installed by default.